.github/workflows/ci.yml

```
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: 14
+          node-version: 16
```

We can actually do: 
```suggestion
          node-version-file: '.nvmrc'
```

Though, now that everyone uses `fnm`, we can rename `.nvmrc` to `.node-version` which is a bit clearer, so the change would then be:
```suggestion
          node-version-file: '.node-version'
```